### PR TITLE
clean up setup instructions

### PR DESCRIPTION
**Pull request by jhoule86**

### CODE REVIEW

File: meta/documents/user_guide/en/initial_setup.md [hunk 1]
```diff
@@ -91,35 +91,33 @@ If Wayfair's shipping services are to be used, the Wayfair plugin's ASN settings
...
 
-9.	In the **Filter** section, a box should appear with a list of all available Order referrers. Place a checkmark next to all **Wayfair** order referrers.
-
+9.	In the `Filter` section, a box should appear with a list of all available Order referrers. Place a checkmark next to all "Wayfair" order referrers:
 ![Wayfair Referrer](https://i.ibb.co/yYpLp8q/asn-04.png "Wayfair Referrer")
 
-10. Click on **Add procedure**, and go to **Plugins >> Send Ship Confirmation (ASN) to Wayfair**. Click on **+ add**.
-
+10. Click on `Add procedure`, and go to `Plugins` >> `Send Ship Confirmation (ASN) to Wayfair`. 
 ![Add Procedure](https://i.ibb.co/xfGrhFP/asn-05.png "Add Procedure")
```
Comment (mcloughlin77) [inline meta/documents/user_guide/en/initial_setup.md:116]: This image has the German translation for "Send Ship Confirmation (ASN) to Wayfair"

File: meta/documents/user_guide/en/initial_setup.md [hunk 2]
```diff
@@ -91,35 +91,33 @@ If Wayfair's shipping services are to be used, the Wayfair plugin's ASN settings
...
 
-The final settings result should look similar to this:
+11. Click on `+ add`.
 
+The final settings result should look similar to this:
 ![Add Procedure Result](https://i.ibb.co/GJPF3ZV/asn-06.png "Add Procedure Result")
```
Comment (mcloughlin77) [inline meta/documents/user_guide/en/initial_setup.md:121]: Similar to the above comment the german plugin name is showing.

File: meta/documents/user_guide/en/initial_setup.md (hunk 1 above)
Comment (jhoule86) [reply meta/documents/user_guide/en/initial_setup.md:116]: Yup. These are from the original docs. For some reason it's a hodge-podge of English and German screenshots. See the new tickets in ProjectHub for supplying separate screenshots for English and German.

File: meta/documents/user_guide/en/initial_setup.md (hunk 2 above)
Comment (jhoule86) [reply meta/documents/user_guide/en/initial_setup.md:121]: Should be fixed when ticket `EM-145` is completed.

**Pull request merged**